readme.md

```diff
@@ -1 +1,86 @@
-Visitor project
+<h1>Visitor project</h1>
+
+The project aims at providing an API for a sample visitor model.
+
+
+<b>Visitor Project Endpoints</b>
+
+<b> GET Request </b>
+**authorisation details to be provided**
+
+url : '/visitor/'
+
+<u> Sample Input </u>
+http -a username:password GET  http://127.0.0.1:8000/visitor/ 
+
+
+<u>Sample Output</u>
+
+HTTP 200 OK
+Allow: GET, POST, HEAD, OPTIONS
+Content-Type: application/json
+Vary: Accept
+
+[
```

Comment:
[ (square brackets ) are present in the output since GET request fetches multiple JSON objects hence creating a JSON array for those objects thus enclosing them in [ ] 
![json](https://cloud.githubusercontent.com/assets/6674956/15739477/dd66bac8-28cd-11e6-8ce0-d8ab9fd52b35.png)